<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRepositoryDialog</class>
 <widget class="QDialog" name="AddRepositoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Repository</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">background-color: #ffffff;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_clone">
            <property name="text">
             <string>Clone (Remote URL required)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_add_existing">
            <property name="text">
             <string>Add existing or initialize...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>628</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="comboBox_search"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_local">
     <property name="title">
      <string>Local</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="DirectoryLineEdit" name="lineEdit_local_path"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_browse_local_path">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_warning">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_remote">
     <property name="title">
      <string>Remote</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_remote_name"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="RepositoryLineEdit" name="lineEdit_remote_url"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_test_repo">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="RemoteAdvancedOptionWidget" name="widget_ssh_override" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_bookmark">
     <property name="title">
      <string>Bookmark</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_bookmark_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RepositoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>RepositoryLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>DirectoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>DirectoryLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>RemoteAdvancedOptionWidget</class>
   <extends>QWidget</extends>
   <header>RemoteAdvancedOptionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioButton_clone</tabstop>
  <tabstop>radioButton_add_existing</tabstop>
  <tabstop>lineEdit_local_path</tabstop>
  <tabstop>pushButton_browse_local_path</tabstop>
  <tabstop>groupBox_remote</tabstop>
  <tabstop>lineEdit_remote_name</tabstop>
  <tabstop>lineEdit_remote_url</tabstop>
  <tabstop>pushButton_test_repo</tabstop>
  <tabstop>groupBox_bookmark</tabstop>
  <tabstop>lineEdit_bookmark_name</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddRepositoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>AddRepositoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
